info.yml

```diff
@@ -18,6 +18,30 @@ configuration:
                      commands that do not match any of the values in the groups
                      setting."
 
+    run_at_startup:
+        type: list
+        description: "Controls what apps will run on startup.  This is a list
+                     where each element is a dictionary with two keys:
+                     'app_instance' and 'name'.
+                     The 'app_instance' value connects this entry to a
+                     particular app instance defined in the environment
+                     configuration file.
+                     The 'name' is the menu name of the command to run when
+                     desktop starts up. If 'name' is '', then all commands from
+                     the given app instance are started.
+
+                     To add the 'Apps' tab (which contains the app launcher),
+                     add the following startup command to the list:
```

Comment:
@robblau Currently, leaving it out does not add the tab. So, if you have the following:

```
- {app_instance: tk-desktop-mytasks, name: ''}
- {app_instance: tk-desktop-mytasks2, name: 'tk-desktop-mytasks2:My Tasks...'}
``` 
No `Apps` tab will show up.

I figured that this would add free flexibility (not sure why you wouldn't want that tab, but it's possible to leave it out anyway) and the `default_value` would take care of the default case that would only use `Apps` and nothing else.

Do you think it would be better if we had a fallback behaviour when the `Apps` selector is not in the config? It felt a bit arbitrary to add this firsthand, but I can see why it could be nice to have the tab added no matter what.